coding/compressed_bit_vector.hpp

```diff
@@ -108,7 +108,7 @@ class DenseCBV : public CompressedBitVector
       for (size_t j = 0; j < kBlockSize; ++j)
       {
         if (((m_bitGroups[i] >> j) & 1) > 0)
-          f(kBlockSize * i + j);
+          f(static_cast<uint32_t>(kBlockSize * i + j));
```

Comment:
|f| usually takes a 32-bit feature id as an argument.
The warning is fixed by either changing the signature of |f| or here. Since feature id is 32 bits, this fix seems better.